styleguide/source/assets/scss/04-templates/_two-column-right--full-width.scss

```diff
@@ -35,4 +35,4 @@
       grid-row: 2 / 2;
     }
   }
-}
+}
```

Comment:
There is no mobile specific breakpoint happening here. And therefore [this line](https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/322/files#diff-e3cc4b1c13088567a514c053188bba96R17) is active on mobile. Meaning that I'm seeing `3fr 1fr`	when I should be seeing full width.